cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,30 @@
+class PaymentStepPage {
+    private BtBankWire: string;
+    private CartNavigation: string
+    private CenterColumn: string
+
+    constructor() {
+        this.BtBankWire = '.bankwire';
+        this.CartNavigation = '.cart_navigation span'
+        this.CenterColumn = '#center_column > div > p > strong'
+
+
+    }
+
+    public PressBankWire(): void {
+        cy.get(this.BtBankWire).click()
+    }
+
+    public PressNavigation(): void {
```

Comment:
This is referring to click on the proceed to checkout button, let's name the method in a way that indicate that we are proceeding to checkout